src/app.main.ts

```diff
@@ -11,18 +9,19 @@ import Path from "path"
 import { Verbosity } from "./types.ts"
 import * as semver from "semver"
 import { VirtualEnv } from "./hooks/useVirtualEnv.ts"
+import { useEnv } from "./hooks/useConfig.ts"
+import { Args } from "./args.ts";
 
 export async function run(args: Args) {
+  const { execPath } = useConfig()
+  const { PATH } = useEnv()
+
   if (args.cd) {
     const chdir = args.cd
     console.verbose({ chdir })
     Deno.chdir(chdir.string)
   }
 
-  if (!Deno.isatty(Deno.stdout.rid) || Deno.env.get('CI')) {
-    logger.set_global_prefix('tea:')
-  }
-
```

Comment:
Is this reproduced somewhere else (didn't see in the diff).

Reply:
It's handled in `useLogger` which pulls from `useConfig` to avoid setting additional global state.

https://github.com/teaxyz/cli/pull/441/files#diff-51338d7c9f90a73701d0b7aac4de25dc47ed8f7d1407ee7f276e17840ea45302R30-R31